### DISCUSSION TITLE
migrating IRC to Libera.Chat

### DISCUSSION THREAD

**Discussion by gour** [GothenburgBitFactory/taskwarrior#2489] [General]
Hello,

any plan to officially migrate IRC channel from Freenode to Libera.Chat?

**Comment by tbabej** [comment]
My understanding is that this is currently underway. I will post here (moving this to the discussions) once it's official.

**Reply by tbabej** [reply]
Taskwarrior channel on libera.chat is now registered and running.

**Comment by ragnarov** [comment]
https://taskwarrior.org/docs/help.html

Please check that page.
It still points to freenode.
Thank you for reading.

**Reply by tbabej** [reply]
Thanks for reporting! Fixed with https://github.com/GothenburgBitFactory/tw.org/commit/02fbdcf0685b7fc185cbabc3970f89f3746ae744

**Comment by djmitche** [comment]
There's also a Discord server for TW.  Should we shut that down?
